shim.js

```diff
@@ -5809,22 +5809,22 @@ dnode.prototype = {};
 function dnode (cons, opts) {
```

Comment:
Did you recompile the js file? I don't get why the extra lines are showing up. They shouldn't be there if you compiled it.